OSDOCS-2925: Note about necessary cluster transfer step in OCM (#38921)
openshift/openshift-docs: modules/images-update-global-pull-secret.adoc

You can update the global pull secret for your cluster by either replacing the current pull secret or appending a new pull secret.

> If you are transferring your cluster to another owner, you must first initiate the transfer in link:https://console.redhat.com/openshift/[Red Hat OpenShift Cluster Manager], then update the pull secret on the cluster. Updating a cluster's pull secret without also initiating the transfer in OpenShift Cluster Manager causes the cluster to stop reporting Telemetry metrics in OpenShift Cluster Manager. See link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-managing-clusters#transferring-cluster-ownership_assembly-managing-clusters[Transferring cluster ownership] in _Managing Clusters_ for instructions.

Reviewer: ```suggestion
To transfer your cluster to another owner, you must first initiate the transfer in link:https://console.redhat.com/openshift/[Red Hat OpenShift Cluster Manager], and then update the pull secret on the cluster. Updating a cluster's pull secret without initiating the transfer in OpenShift Cluster Manager causes the cluster to stop reporting telemetry metrics in OpenShift Cluster Manager. See link:https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021/html/managing_clusters/assembly-managing-clusters#transferring-cluster-ownership_assembly-managing-clusters[Transferring cluster ownership] in "Managing clusters" for instructions.
```
- ISG asks for double quotes around chapter titles. 
- My assumption is that "telemetry metrics" is functioning as a general term rather than a branding-related one.
- Final sentence could also be something like, `For more information [about transferring cluster ownership], see "Transferring cluster ownership" in the Red Hat OpenShift Cluster Manager documentation.`
- Have you considered adding Red Hat OpenShift Cluster Manager to our common attributes file? Should make mass renaming easier in the future. :)

Author: Thanks for the quick feedback!
- Managing Clusters is the title of the OCM guide, should it still be in double quotes?
- Telemetry should be capitalized - it is a service and I've kept it consistent with the rest of the content in the assembly - https://deploy-preview-38921--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/opting-out-of-remote-health-reporting.html#images-update-global-pull-secret_opting-out-remote-health-reporting
- I like your final sentence much better :)
- Great idea with the attribute - I'll create that with the future PR I open for the OCM references.

Also - I just got a late suggestion from an SME to make this a warning or an important box. Can I still make this change?

Reviewer: * I guess it depends on if it is considered a standalone title or a chapter within a docs corpus. Quotes if former, italics if latter. I assume we're talking about the first item here: https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2021
* 👍 
* 👍 
* 👍 

This can be changed as much as you like. If it's significant and you want additional reviewing, just ping the channel again. ✔️
